manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -76,7 +76,9 @@ spec:
             {{- end }}
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
+            # TODO: should this also be .values.linux.providersDir
             - "--provider-volume=C:\\k\\secrets-store-csi-providers"
+            - "--additional-provider-volume-paths={{ .Values.linux.additionalProvidersDirs }}"
```

Comment:
We should also add this new flag and additional path to [linux](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/manifest_staging/deploy/secrets-store-csi-driver.yaml#L57) and [windows](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/manifest_staging/deploy/secrets-store-csi-driver-windows.yaml#L57) yaml by default.